scene/gui/box_container.cpp

```diff
@@ -296,6 +296,7 @@ void BoxContainer::_update_theme_item_cache() {
 
 void BoxContainer::_notification(int p_what) {
 	switch (p_what) {
+		case NOTIFICATION_POST_ENTER_TREE:
```

Comment:
This doesn't seem necessary. The PR still fixes the issue without this line and it's not something other containers do.

Reply:
Yes, it is not necessary. I intend to make the child controls aware of the drawable size early on, but the minimum size needs to be updated at least twice to be determined (after applying #77280).

Reply:
If this line is a part of another issue, it should be removed from this PR.